AyOK-Code/oscn
# Fix address retrieval (issue 10)

### To Test  
make sure oscn is hooked up to the most recent  commit of branch  `jw_Issue_10` in oscn_scraper
run rails console and use these commands from issue 10.
`html = OscnScraper::Requestor::Case.new({county: 'Oklahoma', number: 'CF-2018-1016'}).fetch_c
ase_by_number
parsed = Nokogiri::HTML(html.body)
OscnScraper::Parsers::BaseParser.new(parsed).build_object`


current problem: while address is no longer null, it seems to have some formatting issues.
